File: src/#1/solution.js
var twoSum = function (nums, target) {
  let m = new Map()

  for (let i = 0; i < nums.length; i++) {
    let another = target - nums[i]
    if (m.has(another)) {
      return [m.get(another), i]
    }
    m.set(nums[i], i)
  }
  return null
}

File: src/#1/solution.go
package solution

func twoSum(nums []int, target int) []int {
	m := map[int]int{}

	for i, num := range nums {
		anothor := target - num
		if _, ok := m[anothor]; ok {
			return []int{m[anothor], i}
		}

		m[num] = i
	}

	return nil

}
